File: src/scripts/remove-personal-files.ts
import { expandGlob } from '@std/fs/expand-glob';

for await (const entry of expandGlob('**/*.personal.*', { root: 'src' })) {
    if (entry.isFile) {
        await Deno.remove(entry.path);
        console.log(`Removed: ${entry.path}`);
    }
}

File: src/scripts/remove_personal_files.go
package main

import (
	"io/fs"
	"log"
	"os"
	"strings"
)

func RemovePersonalFiles() {
	fs.WalkDir(os.DirFS("."), ".", func(path string, d fs.DirEntry, err error) error {
		Fatal(err)
		lowerName := strings.ToLower(d.Name())
		if strings.Contains(lowerName, ".personal.") && d.Type().IsRegular() {
			err = os.Remove(path)
			Fatal(err)
			log.Println("Removed: ", path)
		}

		return nil
	})
}
